src/include/net_connect.h

```diff
@@ -172,6 +172,10 @@ enum conn_type {
 	Idle
 };
 
+struct sched_socks {
```

Comment:
Better to have a generic name for this structure as net_server.c is part of the library(Libnet).

Reply:
done